src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/MerchandiseReturnController.php

```diff
@@ -76,11 +91,198 @@ public function indexAction(Request $request, MerchandiseReturnFilters $filters)
         ]);
     }
 
+    /**
+     * @todo so I still have an issue with this not saving properly. Issue is that it's first POST request which redirects to GET request.
```

Comment:
Did'nt we solve this issue in Slack 🤔 ? We said to have 2 differents `path` setting for the routes (the filter POST route and the form submit POST route) in YAML routing file

Reply:
Yeah it's solved, just didn't remove it.